### ISSUE TITLE
[Shorts] Find a better way of finding out if a video is a short

### ISSUE THREAD

**Issue by NikkelM** [NikkelM/Random-YouTube-Video#186]
See #151
The current implementation fails if a video cannot be embedded.
An alternative approach would work like this:
```js
await fetch(`https://www.youtube.com/shorts/${randomVideo}`)
	.then(res => {
		response = res;
	});
```
However, previous tests have shown that this may be a bit slower than the oembed approach. Perhaps both approaches could be bundled, with the second one being the fallback if oembed returns an 'Unauthorized' response.

An alternative approach may be to use a community-driven alternative API which implements a `shorts` endpoint, as described [here](https://stackoverflow.com/questions/71192605/how-do-i-get-youtube-shorts-from-youtube-api-data-v3/71194751#71194751).
It looks like this custom API variant also allows us to fetch only shorts for a given channel, perhaps it also allows us to fetch normal videos and shorts separately, which we could then save as such in the database from the get-go, without having to separate between the two types whenever shuffling.

**Comment by NikkelM** [comment]
If no separate `videos` endpoint is given, we could also fetch both, and remove the intersection of the two from the `videos` list.

**Comment by NikkelM** [comment]
Fixed by #200
